src/Microsoft.Identity.Web.Certificate/DefaultCredentialsLoader.cs

```diff
@@ -10,11 +11,34 @@
 
 namespace Microsoft.Identity.Web
 {
+
     /// <summary>
     /// Default credentials loader.
     /// </summary>
     public class DefaultCredentialsLoader : ICredentialsLoader
     {
+        /// <summary>
+        /// Logging infrastructure
+        /// </summary>
+        private static class Logger
```

Comment:
Please use a similar set up to the high performance logging similar to [TokenAcquisition.Logger.cs](https://github.com/AzureAD/microsoft-identity-web/blob/master/src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.Logger.cs)

Reply:
This has been adressed.